index.css

```diff
@@ -226,135 +236,139 @@ main {
 
 /* Professional project section  */
 .professional__project {
-  background-image: url('./images/section_2.png');
-  width: 327px;
-  height: 386px;
-  margin-bottom: 50px;
+  background-image: url(../icons_copy/cardBackground.svg);
```

Comment:
- [ ] Kindly fix the path on your background image. It should have only one dot. Like ./ at the beginning, in this way, it will be displayed on the browser